stripe/src/main/java/com/stripe/android/paymentsheet/CurrencyFormatter.kt

```diff
@@ -9,19 +9,25 @@ import kotlin.math.pow
 internal class CurrencyFormatter {
     fun format(
         amount: Long,
-        currency: Currency
+        currencyAmount: Currency,
+        targetLocale: Locale = Locale.getDefault()
```

Comment:
maybe `deviceLocale`?

Reply:
I gave this some thought.  The locale here doesn't necessarily need to be the device locale. As this is more of a utility function this might be just the locale you are formatting for.